src/pages/mypage/CompletedMissionDetailPage.tsx

```
@@ -0,0 +1,304 @@
+      <div className="flex w-full items-center gap-2">
+        <span className="heading-3 text-[#424242]">{label}</span>
+        <span className="heading-6 text-[#2664ED]">(객관식)</span>
+      </div>
+      <div className="mt-12 w-full rounded-lg px-12 py-8 text-black">
+        <p className="body-2">{question}</p>
+      </div>
+      <div className="mt-16 flex w-full flex-col gap-8 px-4">
+        {options.map((opt, idx) => (
+          <div
+            key={`${opt}-${idx}`}
+            className="body-2 rounded-lg border border-[#E0E0E0] bg-[#FAFAFA] px-12 py-10 text-black"
```

Biome (2.3.14)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)